trlx/utils/__init__.py

```diff
@@ -86,6 +87,50 @@ def get_distributed_config(accelerator: Accelerator):
     return dist_config
 
 
+class OptimizerNames(Enum):
```

Comment:
Why are we not using registers...?

Reply:
Adding a registry for `str` types seemed like a pre-mature abstraction given that the source is local to a single file. What exactly are you suggesting?

Reply:
Fair, Dw then. Ready for merging imho.